core/gui/src/app/workspace/component/menu/menu.component.html

```diff
@@ -197,32 +197,20 @@
               nzType="comment"></i>
           </button>
           <button
-            (click)="onClickExportExecutionResult('csv')"
-            [disabled]="
-        !workflowResultExportService.exportExecutionResultEnabled ||
```

Comment:
It would be nice to add a method that returns boolean value of 

```
!workflowResultExportService.exportExecutionResultEnabled ||
!workflowResultExportService.hasResultToExportOnAllOperators
```

Reply:
I tried but the problem is `workflowResultExportService.hasResultToExportOnAllOperators` is changing with Observables in `workflowResultExportService` so I have to define a reactive variable for it. That would take more lines of code than repeating these two lines. But if you think its preferred, I make it reactive.

Reply:
Yes. Please make it reactive.